test-runner-poc/config.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+const f = require('util').format;
+const url = require('url');
+const qs = require('querystring');
+const core = require('../lib/core');
+class NativeConfiguration {
+  constructor(environment) {
+    this.options = environment || {};
```

Comment:
Should we call this `this.environment` or instead? It seems a little odd to me that `environment` is becoming `this.options`, but maybe I'm not fully grasping what's going on here.

Reply:
Hmm - there are files in `lib` and `test` that look up `config.options`. I could change those to look up `config.environment`, and get rid of `this.options`?

Reply:
Hmm, let's leave this as-is for now then. Thanks!